jwt.el

```diff
@@ -613,13 +613,58 @@ Specifically it checks that TEST-STRING has
            (doco (car maybe-doc)))
       (funcall callback doco :thing full-name))))
 
+(defvar jwt--annotation-claims
+  '(("nbf" . "Not before")
+    ("iat" . "Issued at")
+    ("exp" . "Expires")))
+
+(defface jwt-annotation
+  '((t
+     (:box
+      (:line-width (1 . 1)
+                   :color nil
+                   :style nil)
+      :inherit (magit-branch-local))))
+  "Face for JWT claim annotation overlays.")
+
+(defun jwt--annotation-add-overlays (beg end)
+  "Add JWT related overlays between BEG and END."
+  (save-excursion
+    (goto-char beg)
+    (let ((claims-rx (rx "\"" (eval (cons 'group (list (cons 'or (mapcar #'car jwt--annotation-claims))))) "\""
+                         (* space) ":" (* space) (group (+ digit)))))
```

Comment:
![1740012937](https://github.com/user-attachments/assets/06357e13-ac8b-4b30-b6ef-612da23d5275)
When the line ends with a comma the overlay is displayed between the numeric date and the comma. IMO it should be at the end of the line. Could be fixed by adding an optional comma to the end of the regexp.